uiowa/uiowa
# Feature: Theming residence hall content type

Resolves https://github.com/uiowa/uiowa/issues/5294. 
Resolves https://github.com/uiowa/uiowa/issues/5373.


## To test:
- `ddev composer install`
- `ddev blt frontend`
- `ddev blt ds --site=housing.uiowa.edu`
- you can use the existing Catlett Hall content that already has fields filled in (https://housing.uiowa.ddev.site/residence-halls/catlett-hall)
- note: for images, floorplans, and 3D view fields you need to upload local files. Using existing media library images will not work correctly.
- Appearance/layout should resemble the mockup (https://sandbox.prod.drupal.uiowa.edu/catlett-hall) for the most part. Notable differences: 
-- The static map is not yet available for the 'Getting around' section. 
-- Section of accordions has changed - Floor plans and 3D view section combined floorplans and room types accordions from before. 
-- Bathrooms accordion replaced by the restroom type section that has its own accordions with photoswipe galleries. 
-- Emergency policy section now vertically displays fields instead of horizontally.

## todo 

- [x]  This is probably going to need to be split patched for sites like hr and admissions
- [x] Create issues for `@todos`
- [x] Add comment that `fivecol` class is meant for images
- [x] Update a few fields to make them required
- [x] `aria-labelledby` on accordions are not being set
- [x] 436 is the nid, but see if you can add the field delta
- [x] fontawesome implementations need role="presentation" 
- [x] Alias pattern be [node:menu-link:parent:url:path]/[node:title] instead of residence-halls/[node:title]
